src/System.Drawing.Common/src/misc/InvariantComparer.cs

```diff
@@ -1,32 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Yeah -- I forgot to mention that. It was internal and nothing was using it. Just a previously-missed cleanup opportunity. Since it was Serializable, I just deleted it.